packages/bff/src/model/domain/apiConverter.ts

```diff
@@ -119,3 +120,9 @@ export function toAuthorizationKeySeed(
     createdAt: new Date().toISOString(),
   };
 }
+
+export const fromApiConsentType = (type: "TOS" | "PP"): PrivacyNoticeKind =>
+  match(type)
+    .with("TOS", () => PrivacyNoticeKind.TOS)
+    .with("PP", () => PrivacyNoticeKind.PP)
+    .exhaustive();
```

Comment:
No need to have this API converter and a customer `PrivacyNoticeKind` type, you could use directly `bffApi.ConsentType.Values.PP` or ``bffApi.ConsentType.Values.TOS` where needed

Reply:
That's correct, it was ported from scala before the big shared API clients refactor :) thanks!